test/integration/discovered_hosts_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'test_plugin_helper'
+require 'integration_test_helper'
+
+class DiscoveredHostsTest < IntegrationTestWithJavascript
+  let(:discovered_host) { FactoryGirl.create(:discovered_host, :with_facts) }
+  let(:discovered_hosts) { Host::Discovered.all }
+
+  setup do
+    discovered_host.save
```

Comment:
I'd perhaps do `save!` so we are sure there is at least one host, I mean if this breaks, all the mass-action tests like reboot might just pass silently.